test/support/coronavirus_helper.rb

```diff
@@ -50,6 +50,12 @@ def content_item_with_live_stream_enabled_and_ask_a_question_enabled
   content_item
 end
 
+def content_item_with_live_stream_enabled_and_popular_questions_link_enabled
+  content_item = coronavirus_content_item_with_live_stream_enabled
```

Comment:
You should set `content_item = coronavirus_content_item` because `coronavirus_content_item_with_live_stream_enabled` is actually no longer a thing. Live stream is always enabled, and the key `live_stream_enabled` isn't on the yaml/content item anymore. 

This has made me realise that the tests need tidying up. Not a blocker for this PR, I'll make a ticket!

Reply:
> You should set `content_item = coronavirus_content_item` because `coronavirus_content_item_with_live_stream_enabled` is actually no longer a thing.

Done! ✅